### ISSUE TITLE
support for qualifiers and references in ShEx

### ISSUE THREAD

**Issue by shigapov** [magnusmanske/cradle#6]
Currently only the ```wdt```-prefix is supported. It would be nice to have support for the following prefixes:
```
PREFIX p: <http://www.wikidata.org/prop/>
PREFIX pq: <http://www.wikidata.org/prop/qualifier/>
PREFIX pr: <http://www.wikidata.org/prop/reference/>
```

**Comment by VDK** [comment]
would be great if the reference support would utilize the Citoid API.

**Comment by magnusmanske** [comment]
Could you give me an example to test on for these?

**Comment by renat-kaufmann** [comment]
[E98](https://www.wikidata.org/wiki/EntitySchema:E98)

**Comment by GreenReaper** [comment]
I agree it'd be nice, as I want to be able to add an optional EDTF point of time to specify start/end time to an optional EDTF period, [as in this example](https://furry.wikibase.cloud/wiki/Item:Q4#P16) ([ShEx](https://furry.wikibase.cloud/wiki/EntitySchema:E2), [cradle](https://furry.wikibase.cloud/tools/cradle/?#/shex/E2)), since EDTF periods can only specified down to the day but events start at a particular time.
